Makefile

```diff
@@ -138,7 +138,7 @@ endif
 
 # Generate bundle manifests and metadata, then validate generated files.
 .PHONY: bundle
-bundle: manifests
+bundle: manifests kustomize
```

Comment:
kustomize should be updated (the 3.5.4 is not really up to date with rest of the projects)

Reply:
Also, generated files used config-gen 0.7.0, but Makefile downloads 0.5.0 - this will cause differences in the CI path also.

Reply:
ok, updating kustomize to 4.0.5 and controller-gen to 0.7.0.

Reply:
done